src/plugins/intel_cpu/src/nodes/mkldnn_adaptive_pooling.cpp

```diff
@@ -68,7 +68,7 @@ void MKLDNNAdaptivePoolingNode::getSupportedDescriptors() {
 
     if (getParentEdges().size() != 2)
         IE_THROW() << errorPrefix << "has incorrect number of input edges: " << getParentEdges().size();
-    if (getChildEdges().size() != (algorithm == AdaptivePoolingMax ? 2 : 1))
+    if (getChildEdges().size() < (algorithm == AdaptivePoolingMax ? 2 : 1))
         IE_THROW() << errorPrefix << "has incorrect number of output edges: " << getParentEdges().size();
```

Comment:
can we have 'getChildEdges()' in throw message instead of getParentEdges()? Also some kind of unit test would be helpful

Reply:
Good catch! Fixed, thanks!
The main problem that all SLT now don't test cases when per one output port we have several output edges. I will update tests for adaptive pooling later